tests/conftest.py

```diff
@@ -0,0 +1,40 @@
+import datetime
+
+import pytest
+
+from cryptography import x509
+from cryptography.hazmat.backends import default_backend
+from cryptography.hazmat.primitives import serialization, hashes
+from cryptography.hazmat.primitives.asymmetric import rsa
+from cryptography.x509 import NameOID
+
+
+@pytest.fixture(name="root_key", scope="session")
+def fixture_root_key():
+    return rsa.generate_private_key(
+        public_exponent=65537,
+        key_size=4096,
+        backend=default_backend())
+
+
+@pytest.fixture(name="rsa_key", scope="session")
+def fixture_rsa_key(root_key):
+    return root_key.private_bytes(
+        encoding=serialization.Encoding.PEM,
+        format=serialization.PrivateFormat.PKCS8,
+        encryption_algorithm=serialization.NoEncryption()).decode("utf-8")
+
+
+@pytest.fixture(name="certificate", scope="session")
+def fixture_certificate(root_key):
```

Comment:
If I understood correctly, we don't actually need to remove .cert files but I thought I could add that too to keep credential usage aligned.